### PR TITLE
move subsystem_info entirely into commander and remove from uORB

### PR DESCRIPTION
**Pull request by dagar**
There's a lot more to do here, but this is an incremental step that gets rid of an unnecessary uORB publication. I'm working towards more detailed status messages for all sensor types that roll into higher level validity checks for commander.

### CODE REVIEW

**Comment by dagar** [comment]
Flash savings 696 Bytes.

``` Console
     VM SIZE    
 -------------- 
  [NEW]    +120    commands.21788
  [NEW]     +52    CSWTCH.323
  +2.3%     +16    Commander::data_link_check()
  +7.4%      +4    IridiumSBD::~IridiumSBD()
  +0.0%      +4    g_cromfs_image
  [ = ]       0    IridiumSBD::csq_loop()
  [ = ]       0    IridiumSBD::publish_iridium_status()
  [ = ]       0    IridiumSBD::read_rx_buf()
  [ = ]       0    ModuleBase<Commander>::main(int, char**)
  [ = ]       0    [Unmapped]
  [ = ]       0    [section .debug_abbrev]
  [ = ]       0    [section .debug_aranges]
  [ = ]       0    [section .debug_frame]
  [ = ]       0    [section .debug_info]
  [ = ]       0    [section .debug_line]
  [ = ]       0    [section .debug_loc]
  [ = ]       0    [section .debug_ranges]
  [ = ]       0    [section .debug_str]
  [ = ]       0    [section .strtab]
  [ = ]       0    [section .symtab]
  -2.2%      -4    IridiumSBD::test_loop()
  -0.6%      -4    uorb_topics_list
  -2.4%      -8    Commander::~Commander()
  -1.3%      -8    IridiumSBD::main_loop(int, char**)
  -0.6%     -16    Commander::Commander()
  [DEL]     -16    __orb_subsystem_info
 -10.3%     -24    IridiumSBD::IridiumSBD()
  [DEL]     -28    int listener_print_topic<subsystem_info_s>(orb_metadata const* const&, int)
  -0.6%     -44    listener_generated(char*, int, int, int)
  [DEL]     -52    CSWTCH.324
  -0.0%     -75    [section .text]
  -1.3%     -84    Commander::run()
  [DEL]     -99    __orb_subsystem_info_fields
  [DEL]    -114    IridiumSBD::publish_subsystem_status()
  [DEL]    -120    commands.21789
  [DEL]    -196    print_message(subsystem_info_s const&)
  -0.0%    -696    TOTAL
```

File: src/modules/commander/Arming/HealthFlags/HealthFlags.h [hunk 1]
```diff
@@ -43,7 +43,40 @@
...
 #include <uORB/topics/vehicle_status.h>
-#include <uORB/topics/subsystem_info.h>
+
+struct subsystem_info_s {
+	// keep in sync with mavlink MAV_SYS_STATUS_SENSOR
+	static constexpr uint64_t SUBSYSTEM_TYPE_GYRO = 1;
+	static constexpr uint64_t SUBSYSTEM_TYPE_ACC = 2;
+	static constexpr uint64_t SUBSYSTEM_TYPE_MAG = 4;
+	static constexpr uint64_t SUBSYSTEM_TYPE_ABSPRESSURE = 8;
+	static constexpr uint64_t SUBSYSTEM_TYPE_DIFFPRESSURE = 16;
```
Comment (MaEtUgR) [inline src/modules/commander/Arming/HealthFlags/HealthFlags.h:53]: Maybe just a detail but I'd define them like this:
```suggestion
	static constexpr uint64_t SUBSYSTEM_TYPE_GYRO = 1<<0;
	static constexpr uint64_t SUBSYSTEM_TYPE_ACC = 1<<1;
	static constexpr uint64_t SUBSYSTEM_TYPE_MAG = 1<<2;
	static constexpr uint64_t SUBSYSTEM_TYPE_ABSPRESSURE = 1<<3;
	static constexpr uint64_t SUBSYSTEM_TYPE_DIFFPRESSURE = 1<<4;
```
More readable, less error-prone.

**Review by MaEtUgR — APPROVED** [review]
A good step! Thanks for the improvement!

**Review by MaEtUgR — APPROVED** [review]
I allowed myself to do the suggestion as an additional commit.

**Comment by dagar** [comment]
> I allowed myself to do the suggestion as an additional commit.

Thanks, the only reason I didn't bother is that eventually we probably won't need these flags as the mapping can be done entirely in the mavlink module. These flags don't completely line up with what we need to do in commander and we shouldn't let Mavlink limit us.

**Pull request merged**